public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -133,6 +134,18 @@ export function ExploreToolbar({ exploreId, topOfViewRef, onChangeTime }: Props)
       <PageToolbar
         aria-label="Explore toolbar"
         leftItems={[
+          config.featureToggles.exploreContentOutline && (
+            <ToolbarButton
```

Comment:
this button should have an `aria-expanded` set to `true/false`  and an `aria-controls="#outliine-container-html-id"` pointing to the id of the root element of the outline when expanded